crates/gpui/src/platform/windows/platform.rs

```diff
@@ -304,9 +304,9 @@ impl Platform for WindowsPlatform {
         }
     }
 
-    // todo(windows)
     fn active_window(&self) -> Option<AnyWindowHandle> {
-        None
+        let active_window_hwnd = unsafe { GetActiveWindow() };
+        try_get_window_inner(active_window_hwnd).map(|inner| inner.handle)
```

Comment:
Because `GetActiveWindow` can return non managed window (like IME popup), `try_get_window_inner` is not safe.

Reply:
Surely the `GWLP_USERDATA` is unset (null) for windows we don't create directly right?

Reply:
No, it depends on creation process of the window. Currentry it is checked using `window_handle_values`.

Reply:
right if someone were to make a window outside of zed **on the same thread** it would be unsafe. I gotcha. I guess that's totally possible. I'll tweak it to not just blindly use the active window result.